inception-project/inception
# #4728 - Leaving the annotation page and coming back when sidebar curation is active leaves the editor in a strange state

**What's in the PR**
- Use the OpenCurationDocument dialog if the sidebar curation mode is active
- Refresh the footer when changing footer items such that we can properly switch between the dialogs and auto-open behaviors
- Avoid error message when trying to destroy an already destroyed Svelte component
- Allow action bar extensions to remove themselves from the footer (needed when we switch between the dialogs)

**How to test manually**
* See issue description
* Try entering/leaving/switch documents, etc. - stress switching between curation and annotation modes in all combinations

**Automatic testing**

**Documentation**
